app/bundles/FormBundle/Translations/en_US/messages.ini

```diff
@@ -54,6 +55,7 @@ mautic.form.field.form.showlabel="Show label?"
 mautic.form.field.form.validationmsg="Validation message"
 mautic.form.field.group.labelattr="Radio/checkbox label attributes"
 mautic.form.field.help.attr="Add custom HTML attributes i.e. class=&quot;myclass&quot;"
+mautic.form.field.help.auto_fill = "Enable auto fill, will be auto fill data lead field to form"
```

Comment:
Let's reword this to something like "Enable to auto fill value from the tracked lead's profile."